ios-10-0-arm64-dep-8-0-hid-sections.cmake

```diff
@@ -29,7 +29,7 @@ include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
 include(polly_fatal_error)
 
 # Fix try_compile
-set(MACOSX_BUNDLE_GUI_IDENTIFIER com.example)
+set(MACOSX_BUNDLE_GUI_IDENTIFIER com.example.polly)
```

Comment:
Please move this to separate module. In this module we should read environment variable `POLLY_IOS_BUNDLE_IDENTIFIER`. If this variable is empty then use `com.example`. If it's not empty do `set(MACOSX_BUNDLE_GUI_IDENTIFIER ${POLLY_IOS_BUNDLE_IDENTIFIER})`.